tests/benchmark/spark/5_generate_results.sh

```diff
@@ -4,6 +4,9 @@ source PerfRun.conf
 directory=$outputLocation/$(date "+%Y.%m.%d-%H.%M.%S")_$dataSize$queries
 mkdir -p  $directory
 
+mv *.csv *.out $directory/
```

Comment:
Also copy csv files (we now write the timings to CSV files for better structure)